src/app/list/basic-list/list.component.ts

```diff
@@ -152,7 +152,17 @@ export class ListComponent extends ListBase implements DoCheck, OnInit {
     item.expanded = false;
   }
 
-  private toggleExpandArea(item: any): void {
+  /**
+   * Toggles the list item expansion
+   * 
+   * @param {MouseEvent} $event The event emitted when an item has been clicked
+   * @param {Object} item The object associated with the current row
+   */
+  private toggleExpandArea($event: any, item: any): void {
```

Comment:
I believe the event type should be `MouseEvent`. We don't want to use type `any` when possible -- `item` is an exception.